lib/ProductOpener/DataQualityFood.pm

```diff
@@ -758,6 +761,37 @@ sub check_nutrition_data ($product_ref) {
 			if (($nid eq 'fat') or ($nid eq 'carbohydrates') or ($nid eq 'proteins') or ($nid eq 'salt')) {
 				$total += $product_ref->{nutriments}{$nid . "_100g"};
 			}
+
+			# variables to check if there are 3 or more duplicates in nutriments
+			if (
+				(
+					   ($nid eq 'fat_100g')
+					or ($nid eq 'saturated-fat_100g')
+					or ($nid eq 'carbohydrates_100g')
+					or ($nid eq 'sugars_100g')
+					or ($nid eq 'fiber_100g')
+					or ($nid eq 'proteins_100g')
+					or ($nid eq 'salt_100g')
+					or ($nid eq 'sodium_100g')
```

Comment:
I would remove sodium, which is computed from salt. Otherwise we may get a lot of false positives when salt is 0 (thus sodium will be 0 too) and anything else is 0 (fat, sugar etc.)

Reply:
ah I spoke too fast, there's a check > 1 after, so salt and sodium will be different. That's fine then.